examples/Handpose-p5-start-stop/sketch.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+let handpose;
+let video;
+let hands = [];
+let isDetecting = false;
+
+function preload() {
+  // Load the handpose model.
+  handpose = ml5.handpose();
+  console.log("preload");
```

Comment:
I would remove this or change it to "Hand Pose model is loaded." or something like that?

Reply:
Ah, this was for testing and I totally forgot to remove it. Thank you for pointing it out, removing now!